### PR TITLE
Rs 2023.05.21

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: system_strings/sysmes_text.en [hunk 1]
```diff
@@ -6501,47 +6550,71 @@ One week has passed since that^night. Within my unchanging^daily life, I remembe
...
+// @flowchart_descr
+I pierce her with my knife.
+ I pray that those scars are mine alone to bear―――
 }
 [sha:b75f12c5fc4b10f3143bdf364f5969ae08628f5f]{
 -- この命はじきに停止する。^だが、ロアの自由にはさせない。^残された方法はただひとつ。
-My life will soon be at an end.^But I can't allow Roa to roam^free――there is only one thing^left to do.
+// @flowchart_descr
+My life will soon be at an end.
+ But I can't allow Roa to roam free――there is only one thing left to do.
```
Comment (Petrikow) [inline system_strings/sysmes_text.en:6586]: first line is a modification of 38257
no changes necessary here, but maybe "this body" to "my body" there

File: system_strings/sysmes_text.en [hunk 2]
```diff
@@ -6501,47 +6550,71 @@ One week has passed since that^night. Within my unchanging^daily life, I remembe
...
 }
 [sha:f788a3d127b76c2c93e5a3c7779e64f976641378]{
 -- 彼女を嫌いにはなれない。^……だからこそ、^その間違いは見逃せない。
-I can't bring myself to hate^her... But that's also why I^can't overlook this mistake.
+// @flowchart_descr
+I can't bring myself to hate her... But that's also why I can't overlook this mistake.
```
Comment (Petrikow) [inline system_strings/sysmes_text.en:6591]: ref 38316/38317 (we swapped a bit there)

>...
>Which is why I could never let her get away with making such a mistake.

File: system_strings/sysmes_text.en [hunk 3]
```diff
@@ -6501,47 +6550,71 @@ One week has passed since that^night. Within my unchanging^daily life, I remembe
...
+// @flowchart_descr
+Arcueid reveals her true power.
+ Ciel and I retreat to the school, and ready ourselves for battle.
 }
 [sha:672cdb4904185359fd8b1f30738b33c7fb68b3ea]{
 -- 第七聖典による狙撃を^敢行するシエル。^渾身の一射は、はたして。
-Ciel uses the Seventh Scripture^to snipe her target. Her first^shot, with all her strength^behind it, goes as expected.
+// @flowchart_descr
+Ciel uses the Seventh Scripture to snipe her target.
+ Her first shot, with all her strength behind it, goes as expected.
```
Comment (Petrikow) [inline system_strings/sysmes_text.en:6604]: はたして misinterpreted

>...
>Will her mighty shot strike down her foe?

it's that kinda nuance

File: system_strings/sysmes_text.en [hunk 4]
```diff
@@ -6501,47 +6550,71 @@ One week has passed since that^night. Within my unchanging^daily life, I remembe
...
 }
 [sha:14bff6ca7cbc7552d9313f4d39e210febb6c17e8]{
 -- 街が熔け、消える。^これは自分と志貴だけの世界。^なのに邪魔者が混ざっている――
-The city melts away and^vanishes. A world for just^Shiki and me. And yet, an^intruder has entered the mix.
+// @flowchart_descr
+The city melts away and vanishes.
+ A world for just Shiki and me.
+ And yet, an intruder has entered the mix.
```
Comment (Petrikow) [inline system_strings/sysmes_text.en:6611]: ref 39061 & 39062

39062 could probably have 'had' removed just so it works as a continuous sentence better (it'd be okay on its own, but... context)

File: system_strings/sysmes_text.en [hunk 5]
```diff
@@ -6501,47 +6550,71 @@ One week has passed since that^night. Within my unchanging^daily life, I remembe
...
 }
 [sha:203ae576ab40b4d5904b38b6fad776fd57a72777]{
 -- 校舎の屋上へと駆け上がる。^夜を照らす閃光の中、^内から語りかける声を聞く。
-I rush up to the roof of the^school. As a flash of light^illuminates the city, I listen^to the voice inside my head.
+// @flowchart_descr
+I rush up to the roof of the school.
+ As a flash of light illuminates the city, I listen to the voice inside my head.
```
Comment (Petrikow) [inline system_strings/sysmes_text.en:6617]: It's more like this
>...
>As a flash of light illuminates the city, **I hear a voice start talking to me inside my head.**

File: system_strings/sysmes_text.en [hunk 6]
```diff
@@ -6501,47 +6550,71 @@ One week has passed since that^night. Within my unchanging^daily life, I remembe
...
 }
 [sha:e0e22a7f7ff19b9f9182de198745261c252afa05]{
 -- 完全武装シエル、対、^真祖アルクェイド。^勝利の星はどちらに輝く。
```
Comment (Petrikow) [inline system_strings/sysmes_text.en:6620]: The Luminary appears > The Luminary manifests.

Just to be consistent with the BGM of the same title.

File: system_strings/sysmes_text.en [hunk 7]
```diff
@@ -6570,15 +6643,23 @@ The Luminary appears.
...
+ Who cares if this is my last chance to turn back?
+ Retreat isn't an option.
 }
 [sha:43a21951246fb02775084d4a9c12b0aaf88eebf4]{
 -- ロアは語る。^勝ち目が存在しないのなら、^何を“勝ち”とするべきかを。
-ロアは語る。^勝ち目が存在しないのなら、^何を“勝ち”とするべきかを。
+// @flowchart_descr
+Roa speaks.
+ If I have no way of seizing victory,
+ then what should I consider to be my victory condition?
```
Comment (Petrikow) [inline system_strings/sysmes_text.en:6662]: >Roa explains.
>...
>...

probably matches the context better

File: system_strings/sysmes_text.en [hunk 8]
```diff
@@ -6654,7 +6742,9 @@ And yet those feelings will never be conveyed.
 }
 [sha:7688259652dad5489cc424b5d0fb6114b7a90c4e]{
 -- 愚者の行いは^時に賢者すら堕落させる。^ましてや歓びを与えるなど。
-愚者の行いは^時に賢者すら堕落させる。^ましてや歓びを与えるなど。
+// @flowchart_descr
+Even the wise may be corrupted by the deeds of fools.
+ Especially when those deeds bring joy.
```
Comment (Petrikow) [inline system_strings/sysmes_text.en:6747]: It's probably this
>...
>Or worse, they may even bring them joy.

File: system_strings/sysmes_text.en [hunk 9]
```diff
@@ -6678,23 +6768,39 @@ I reconcile with the vampiress, and say my farewells.
 }
 [sha:10d6c3f49e50556e0524542a0731f28254403ca1]{
 -- 鈍器で殴られるような頭痛。^この部屋の外に未来はない。^この部屋にいれば命がない。
-鈍器で殴られるような頭痛。^この部屋の外に未来はない。^この部屋にいれば命がない。
+// First line in the text (37059) kinda split context with 37060.
+// Last line kinda like 37129.
+// @flowchart_descr
+My head is pounding.
+ There's no future for me outside this room.
+ If I stay in here, I'll die.
```
Comment (Petrikow) [inline system_strings/sysmes_text.en:6776]: Should probably reflect the contrastive nature
>...
>...
>But if I...

File: system_strings/sysmes_text.en [hunk 10]
```diff
@@ -6678,23 +6768,39 @@ I reconcile with the vampiress, and say my farewells.
...
 }
 [sha:819b410c2eae5c24a8cd68f52a8c3809324d2887]{
 -- 現れる真祖の姫。^幾度めの逢瀬は^甘いものではなく。
-現れる真祖の姫。^幾度めの逢瀬は^甘いものではなく。
+// @flowchart_descr
+The princess of the True Ancestors appears.
+ Our umpteenth tryst is not so sweet.
```
Comment (Petrikow) [inline system_strings/sysmes_text.en:6782]: Also contrast, lol
>...
>But our...

File: system_strings/sysmes_text.en [hunk 11]
```diff
@@ -6678,23 +6768,39 @@ I reconcile with the vampiress, and say my farewells.
...
 }
 [sha:18e2eb03c2b1b41c5929950f462ad5028f6cf8ac]{
 -- 爪とナイフがぶつかり合う。^吸血鬼と化した体は躍動する。^だが、この女の線は見えない――
-爪とナイフがぶつかり合う。^吸血鬼と化した体は躍動する。^だが、この女の線は見えない――
+// @flowchart_descr
+Knife clashes with claws.
+ My vampirized body moves like a charm.
+ But, I can't see her lines at all――
```
Comment (Petrikow) [inline system_strings/sysmes_text.en:6789]: dash not needed

File: system_strings/sysmes_text.en [hunk 12]
```diff
@@ -6678,23 +6768,39 @@ I reconcile with the vampiress, and say my farewells.
...
+Life goes on.
 }
 [sha:cd67381c5d3a067b5098323b6c81c4149eff24da]{
 -- 亡くした幼い時分の思い出。^積み上げた今までの思い出。^―――ふと、目が覚めた。
-亡くした幼い時分の思い出。^積み上げた今までの思い出。^―――ふと、目が覚めた。
+// Nice callback to the first line of the prologue prologue
+// @flowchart_descr
+Memories of a long-lost childhood.
+ Memories of all that has happened.^^
+Suddenly, I wake up.
```
Comment (Petrikow) [inline system_strings/sysmes_text.en:6803]: omg I knew I recognized that line
ふと、目が覚めた is the first line of Tsukihime (Offset 2560)

File: system_strings/sysmes_text.en [hunk 13]
```diff
@@ -1332,6 +1335,7 @@ Primordial Earth
 }
 [sha:96e8a9474e1d4f40ad7ecd265762021e84c2beee]{
 -- 逆行運河／天体受胎
+// @flowchart_title
 Inverse Channel/Celestial Birth
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply system_strings/sysmes_text.en:1339]: Afraid that runs heavily afoul of the line limit
2023-05-31-204907_794x154_scrot

File: system_strings/sysmes_text.en (hunk 8 above)
Comment (rschlaikjer) [reply system_strings/sysmes_text.en:6747]: Subj/obj a bit confusing as suggested (the wise would bring the deeds joy?). Suggest
> Or worse, made to feel joy.

File: system_strings/sysmes_text.en (hunk 12 above)
Comment (rschlaikjer) [reply system_strings/sysmes_text.en:6803]: closing the circle

File: system_strings/sysmes_text.en (hunk 13 above)
[the remark this answers is not in the document]
Comment (Petrikow) [reply system_strings/sysmes_text.en:1339]: Guess we'll have to run with MBAACC's localization

>Retroflow/Celestial Conception

File: system_strings/sysmes_text.en (hunk 8 above)
Comment (Petrikow) [reply system_strings/sysmes_text.en:7249]: No, like this
True that it was unclear though

>Or worse, those deeds may even bring them (the wise) joy.

**Pull request merged**